Dockerfile

```diff
@@ -32,11 +32,10 @@ RUN set -x \
  && /fossology/install/scripts/php-conf-fix.sh --overwrite \
  && /fossology/utils/fo-installdeps -e -y \
  && $_cleanup
-RUN curl -sS https://getcomposer.org/installer | php && \
-    mv composer.phar /usr/local/bin/composer
 
 ADD . .
 RUN chmod +x /fossology/docker-entrypoint.sh
+RUN make composer_download && make composer_install
```

Comment:
The `composer_install` target is also called later when `make install` is called (see: https://github.com/fossology/fossology/blob/ae754b26e849b4c9068ed6f4aa354310165aec0c/src/Makefile#L53)